### PR TITLE
Add test for timestamps retrieval

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
Add test to ensure that timestamp retrieval works as intended with different timezone offsets.

Related Issue: 
https://github.com/ballerina-platform/ballerina-standard-library/issues/129

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [x] Added tests

### CODE REVIEW

File: ballerina/tests/timestamp-test.bal [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+    test:assertEquals(check datetimetzInoutValue.get(time:Utc), check time:utcFromString(datetimetz),
+                      "Retrieved date time with timestamp does not match.");
+}
+
+@test:Config {
+    groups: ["timestamp"]
+}
+public function testStringParamTimestamp() returns error? {
+    Client dbClient = check new (host, user, password, database = proceduresDatabase);
```
Comment (niveathika) [inline ballerina/tests/timestamp-test.bal:48]: Write a util class which will take an initialised client and execute the query and returns the procedurecallresult

File: ballerina/tests/timestamp-test.bal [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+// KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/sql;
+import ballerina/time;
+
+@test:Config {
+    groups: ["timestamp"]
```
Comment (niveathika) [inline ballerina/tests/timestamp-test.bal:21]: Lets reorganise and add these test cases within `procedures` group

Comment (kaneeldias) [reply ballerina/tests/timestamp-test.bal]: Fixed in 0479e8d

File: ballerina/tests/timestamp-test.bal (hunk 1 above)
Comment (kaneeldias) [reply ballerina/tests/timestamp-test.bal]: Fixed in 0479e8d

**Review by niveathika — APPROVED** [review]

**Pull request merged**